### PR TITLE
Moving to brew iOS-webkit-debug-proxy

### PR DESCRIPTION
**Pull request by shridharkalagi**
Use ios-webkit-debug-proxy from brew for reliable tests

### CODE REVIEW

**Comment by SrinivasanTarget** [comment]
@shridharkalagi Can you add in Readme that IWDP needs to be installed explicitly.

**Comment by shridharkalagi** [comment]
@SrinivasanTarget @saikrishna321 Pre requisites are in wiki. Can you update it as I dont have access? 

`ios-webkit-debug-proxy` (aka iwdp) should be installed only if you are running tests on Safari browser.

To install iwdp - `brew install ios-webkit-debug-proxy`

File: src/main/java/com/appium/ios/IOSDeviceConfiguration.java [hunk 1]
```diff
@@ -154,23 +152,8 @@ public boolean checkiOSDevice() throws Exception {
...
-        try {
-            port = ap.getPort();
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
-        String webkitRunner =
-                ios_web_lit_proxy_runner + " -c " + DeviceManager.getDeviceUDID()
-                        + ":" + port + " -d";
-        System.out.println(webkitRunner);
+        String webkitRunner = "ios_webkit_debug_proxy -c " + DeviceManager.getDeviceUDID() + ":"
```
Comment (SrinivasanTarget) [inline src/main/java/com/appium/ios/IOSDeviceConfiguration.java]: @shridharkalagi Can we somehow throw error if user doesnt install iwdp or incase if iwdp doesn't start properly?

**Comment by SrinivasanTarget** [comment]
> Pre requisites are in wiki

Yeah i forget that, we can update that.

**Comment by shridharkalagi** [comment]
Creating PR to a different branch

**Pull request closed without merging**